pinot-spi/src/main/java/org/apache/pinot/spi/annotations/ScalarFunction.java

```diff
@@ -46,5 +46,10 @@
 
   boolean enabled() default true;
 
-  String name() default "";
+  // A comma-separated function names to register in FunctionRegistry
+  String names() default "";
```

Comment:
We can keep it as `name()` to be backward compatible?

Reply:
There is no usage of this `name()`, but could be usage from pinot users.

I can add it back and mark deprecated.